build-logic/src/main/kotlin/kotlinx/io/conventions/kotlinx-io-publish.gradle.kts

```diff
@@ -68,22 +68,12 @@ fun MavenPublication.mavenCentralArtifacts(project: Project, sources: SourceDire
     artifact(javadocJar)
 }
 
-
-fun mavenRepositoryUri(): URI {
-    val repositoryId: String? = System.getenv("libs.repository.id")
-    return if (repositoryId == null) {
-        URI("https://oss.sonatype.org/service/local/staging/deploy/maven2/")
-    } else {
-        URI("https://oss.sonatype.org/service/local/staging/deployByRepositoryId/$repositoryId")
-    }
-}
-
-fun RepositoryHandler.configureMavenPublication( project: Project) {
+fun RepositoryHandler.configureMavenPublication(project: Project) {
     maven {
-        url = mavenRepositoryUri()
+        url = uri(project.getSensitiveProperty("libs.repo.url")!!)
```

Comment:
```suggestion
        url = uri(project.getSensitiveProperty("libs.repo.url") ?: error("libs.repo.url is not set") )
```

Reply:
Updated it to simply ignore repository setup if the url is not provided (which is always the case, except running our CI tasks)